hat/app/org/hatdex/hat/utils/NetworkRequest.scala

```diff
@@ -0,0 +1,60 @@
+package org.hatdex.hat.utils
```

Comment:
The `object` is specific to the particular contract logic. As such it should be moved to the contract module.